hack/generator/pkg/codegen/pipeline_apply_export_filters.go

```diff
@@ -0,0 +1,45 @@
+package codegen
+
+import (
+	"github.com/Azure/k8s-infra/hack/generator/pkg/astmodel"
+	"github.com/Azure/k8s-infra/hack/generator/pkg/config"
+	"k8s.io/klog/v2"
+)
+
+func applyExportFilters(configuration *config.Configuration) PipelineStage {
```

Comment:
minor: Since this is actually construction an object (and not "doing" anything), wonder if it would be better to phrase the method like:
`newApplyExportFiltersPipelineStage` or something, rather than as an action like it is now.

The same question applies below to the `deleteGeneratedCode` method

Reply:
I chose this method name to make the pipeline more declarative, so the naming does look a bit odd in this location; don't want to change it, but will add a comment to clarify.

Reply:
Done.